go.mod

```diff
@@ -1,5 +1,5 @@
 module github.com/pelletier/go-toml/v2
 
-go 1.15
+go 1.16
```

Comment:
The go version should stay at 1.15 because go-toml supports the last two versions of go. It will be bumped to 1.16 when 1.17 is released.

Reply:
makes sense, I wasn't aware of that